lowfi_simulation/src/main/native/include/MotorModel/LinearMotorModelSimulation.h

```diff
@@ -0,0 +1,40 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include "MotorModel/MotorModelSimulation.h"
+
+namespace frc {
+namespace sim {
+namespace lowfi {
+
+class LinearMotorModelSimulation : public MotorModelSimulation {
+ public:
+  explicit LinearMotorModelSimulation(double maxSpeed);
+  ~LinearMotorModelSimulation();
+
+  virtual void Reset();
+  virtual void SetVoltage(double voltage);
+  virtual void Update(double elapsedTime);
+
+  virtual double GetPosition();
+  virtual double GetVelocity();
+  virtual double GetAcceleration();
+
+ protected:
+  double m_voltagePercentage;
```

Comment:
Instead of using the constructor initializer list to initialize these with constants, initialize them like `double m_maxSpeed = 0;`.

Reply:
Believe it or not, at work I still have to use C++98 most of the time, and I'm not used to this fancy, modern day stuff

Reply:
That sounds painful...